frontend/src/app/libs/common-components-demo/components/popover-common-demo/popover-common-demo.component.html

```diff
@@ -0,0 +1,48 @@
+<app-demo-group class="button-demo" [title]="'Common Popover'">
+	<div class="light-theme button-demo__states">
+        <app-popover-common #templatepopover >
+			<div class="popover-title">
+				Title
+			</div>
+			<div class="popover-content">
+				Content Lorem ipsum
+			</div>
+			<div class="popover-actions">
+				<button mat-button (click)="templatepopover.close()"> Action</button>
+			</div>
+            <ng-container *ngTemplateOutlet="demoinputtemplate" ngProjectAs="[foo]"></ng-container>
+        </app-popover-common>
+		<div class="button-demo__active">
+			<app-demo-state [title]="'light'" class="button-demo__container">
+				<app-button type="flat" (click)="templatepopover.open()">Button</app-button>
+			</app-demo-state>
+		
+		</div>
+
+	</div>
+	<div class="dark-theme button-demo__states">
+            <app-popover-common [theme]="darkTheme" #templatepopoverdark>
+				<div class="popover-title">
+					Title
+				</div>
+				<div class="popover-content">
+					Content Lorem ipsum
+				</div>
+				<div class="popover-actions">
+					<button mat-button (click)="templatepopoverdark.close()"> Action</button>
+				</div>
+                <ng-container *ngTemplateOutlet="demoinputtemplate" ngProjectAs="[foo]"></ng-container>
+			</app-popover-common>
+		<div class="button-demo__active">
+			<app-demo-state [title]="'darks'" class="button-demo__container" [isDark]="true">
+				<app-button type="flat" [theme]="darkTheme" (click)="templatepopoverdark.open()">Button</app-button>
+			</app-demo-state>
+		</div>
+	</div>
+</app-demo-group>
+
+<ng-template #demoinputtemplate>
+        <div class="popover-title">Title</div>
```

Comment:
not use why we need this ng template, seems like it duplicates logic that already exists